projects/kit/components/input-phone-international/input-phone-international.template.html

```diff
@@ -37,6 +37,8 @@
         (ngModelChange)="onValueChange($event)"
     />
 
+    <ng-content select="tui-icon" />
```

Comment:
`<ng-content select="tui-icon, img" />` Can I put my image source?

Reply:
@waterplea I had the same thoughts too.
Probably, I should add `TODO` (for `5.0`)

Reply:
@splincode fixed in https://github.com/taiga-family/taiga-ui/pull/8750/commits/a4d1e45baa2ae3bd562964e57a618701694caf85

@waterplea fixed in https://github.com/taiga-family/taiga-ui/pull/8750/commits/baece221754662f6bd9793fdf3e68921fc299e59